awx_collection/README.md

```diff
@@ -16,7 +16,7 @@ This Ansible collection allows for easy interaction with an AWX server via Ansib
 
 This source for this collection lives in the `awx_collection` folder inside of the
 AWX source.
-The previous home for this collection was inside the folder [lib/ansible/modules/web_infrastructure/ansible_tower](https://github.com/ansible/ansible/tree/stable-2.9/lib/ansible/modules/web_infrastructure/ansible_tower) in the Ansible repo,
+The previous home for this collection was inside the folder [lib/ansible/modules/web_infrastructure/ansible_controller](https://github.com/ansible/ansible/tree/stable-2.9/lib/ansible/modules/web_infrastructure/ansible_controller) in the Ansible repo,
```

Comment:
Should all instances of "Ansible Controller" actually be "Automation Controller"? I might not know what was agreed upon for this particular renaming, but in other places I've seen `Ansible Tower` `--->` `Automation Controller`.

Reply:
Correct, it's automation controller.